cg/models/demultiplex/flow_cell.py

```diff
@@ -45,7 +50,8 @@ def __init__(self, flow_cell_path: Path, bcl_converter: Optional[str] = None):
         self.position: Literal["A", "B"] = "A"
         self.parse_flow_cell_dir_name()
         self.bcl_converter: Optional[str] = self.get_bcl_converter(bcl_converter)
-        self._sample_sheet_path_hk: Optional[Path] = None
+        self.sample_sheet_path_hk: Optional[Path] = sample_sheet_path
```

Comment:
```suggestion
        self.sample_sheet_path: Optional[Path] = sample_sheet_path
```
Not sure whether we should care if it is from housekeeper here.

Reply:
The name clashes with 
```
    @property
    def sample_sheet_path(self) -> Path:
        """
        Return sample sheet path.
        """
        return Path(self.path, DemultiplexingDirsAndFiles.SAMPLE_SHEET_FILE_NAME.value)
```
I renamed this one to `sample_sheet_run_dir_path`